change/@internal-react-components-3f6426b0-9fb2-4856-aff3-40edc7924dfb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "add id",
```

Comment:
Update comment so it can be used in our release changelog. Added what id to what?

I also see:
![image](https://user-images.githubusercontent.com/2684369/162259983-6648e855-dc64-4a49-8f6f-0d2aed936b37.png)

Which looks like a functional change we should include in the changelog